website/src/categories/getting-started/developers.md

```diff
@@ -99,10 +123,42 @@ This will load the button component and register it as `<sl-button>`. After that
 
 ## Example project
 
-To serve as an inspiration (or just something to copy-paste from) we've made an example project that uses several Design System components so you can study how they interact or have something to compare your code to if something doesn't work as expected.
+To serve as an inspiration (or just something to copy-paste from) we've made some example projects that uses several Design System components so you can study how they interact or have something to compare your code to if something doesn't work as expected.
 
-You can find the [example project repository on Github](https://github.com/sl-design-system/example-design-system-lit-app)
+You can find the [Angular example project repository on Github](https://github.com/sl-design-system/angular-demo)
 
-</div>
+You can find the [Lit example project repository on Github](https://github.com/sl-design-system/example-design-system-lit-app)
+
+</section>
+
+<section>
+
+## Troubleshooting and FAQ
+
+### The font in the components doesn't match the rest of my application.
+
+Make sure the font you use in the application has the exact same name as the font used in the theme tokens. We took an inventatisation on which themes use which font, so in theory it should work out of the box. If that is not the case change the name of your font family name in your app.
+Because the font name in the token need to match the name of the font used in Figma we are a little less flexible.
+If the names match but the font still doesn't load check if the font is actually loaded. The components use the font available in the app, so if you need to add fonts do this like you normally would.
+
+
+### How do I use the dark mode of the theme?
+
+This only applies to themes that support dark mode. 
+When you include the `all.css` file both light and dark mode are included and triggered by the system or browser preference (using `@media (prefers-color-scheme: dark)`). If you want to control the use of the light and dark mode you can load the `light.css` and `dark.css` based on the condition you want. Don't forget to also include `base.css` in addition to the light and dark file. You won't need `all.css` when using the separate files.
+
+
+### How do I make sure my build pipeline has access to the packages as well?
```

Comment:
- For Bitbucket, SL TechOps has set up a token for the pipelines
- Itslearning has set up integration internally
- Magister has set up integration internally for Azure Devops